config/sync/block.block.exposedformhealth_conditionssearch_page_2.yml

```diff
@@ -0,0 +1,31 @@
+uuid: 371460b8-b03d-4f30-9747-1fa6d087f98d
+langcode: en
+status: true
+dependencies:
+  config:
+    - views.view.health_conditions
+  module:
+    - ctools
+    - views
+  theme:
+    - nicsdru_nidirect_theme
+id: exposedformhealth_conditionssearch_page_2
```

Comment:
I would aim to replace the generic `_page_2` suffix with something more descriptive to make it clearer in future what this block filter is for. Probably more relevant for the original view/display definition rather than the exposed form block as that by itself is fairly obvious.

Reply:
same comments apply as above for the `block.block.healthconditionsatoz_2.yml` - the block appears at different places dependent upon the type of page being viewed. It would be sensible to keep the naming of the blocks consist to make it clear what purpose they perform, are you content that the suffix takes the form _side_2?